scripts/exhafs_grid.sh

```diff
@@ -259,6 +281,54 @@ fi
   cp $filter_dir/oro.${CASE}.tile${tile}.shave.nc $out_dir/${CASE}_oro_data.tile${tile}.halo${halo0}.nc
   cp $filter_dir/${CASE}_grid.tile${tile}.shave.nc  $out_dir/${CASE}_grid.tile${tile}.halo${halo0}.nc
 
+
```

Comment:
@BantwaleEnyew-NOAA Could you please revert these changes, which should not be part of this PR? Please also double check to use /bin/sh instead of /bin/ksh for this script. Thanks!